### ISSUE TITLE
Некоторые курсы могут быть только оффлайновыми

### ISSUE THREAD

**Issue by gambala** [Freika/ruby_courses#34]
Например RubyGarage (http://rubycourses.ru/courses/41). Только оффлайн и только в Днепре. Возможно стоит добавить возможность как-то указывать это на сайте?

**Comment by Freika** [comment]
@gambala да, я планирую в конце января-феврале обновить проект, и добавлю это в том числе, спасибо
